---
title: "People"
Description: " "
featured_image: './images/hexagons2.png'
---
# Meet our group members!
</br>

## Group leader

<h4 id="wouter"></h4>

![](/images/Wouter.png) <br />
#### Wouter Boomsma<br><span style="font-weight:normal">(professor)</span>
<details>
<summary> <strong><em>Main interest (click to expand):</em></strong> <br />
Development of Machine Learning techniques for solving fundamental challenges in biology.<br />
</summary>
<br />
  Currently, the strongest focus in my group is on models for understanding the relationship between protein sequence, structure and function. This includes representation learning of protein sequence and structure, prediction of 3D structure, and systematic optimization of proteins for specific traits. Our methods have applications in diverse areas, both in industry (protein engineering), basic biology (how do mutations affect protein interaction networks) and health (understanding the role of mutations in human disease).
</details>

</br> | | | | | |
:-:|:-:|:-:|:-:|:-:|:-:|
| ![](/social/half_filler.png) |[![](/social/twitter.png)](https://twitter.com/WouterBoomsmaDK) | [![](/social/email.png)](mailto:wb@di.ku.dk) |[![](/social/website.png)](https://di.ku.dk/english/staff/vip/?pure=en/persons/275482)  | [![](/social/linkedin.png)](https://www.linkedin.com/in/wouter-boomsma-80b1376) | ![](/social/half_filler.png) |

<br />

## Current group members

<h4 id="simon"></h4>

![](/images/Simon.png) <br />
#### Simon Bartels<br><span style="font-weight:normal">(postdoc)</span>
<details>
<summary> <strong><em>Main interests (click to expand):</em></strong> <br />
Gaussian process inference and optimization of protein functions. <br />
</summary>
<br />
On the practical side, I want to improve the protein design workflow. With Bayesian optimization, I hope to reduce the number of expensive and time consuming wetlab experiments. </br>
Using results from preceding experiments, unlabelled data from related tasks and simulations, should allow to make informed choices about which protein modifications are worthwhile to explore.</br>
On the theoretical side, I am pursuing the question: how much of this data is necessary to obtain a decent (Gaussian process) model.</br>
Importantly, I want an answer for the dataset at hand and NOT for all datasets.</br>
For solving this problem, I am interested in optimal stopping and probably-approximately-correct bounds.
</details>

</br> | | | | |
:-:|:-:|:-:|:-:|:-:|
| ![](/social/filler.png) |![](/social/filler.png) |[![](/social/website.png)](https://simonbartels.github.io/)  | ![](/social/filler.png) | ![](/social/filler.png) |

<br /><br />


<h4 id="marloes"></h4>

![](/images/Marloes.png)
#### Marloes Arts <br><span style="font-weight:normal">(PhD student)</span>
<details>
<summary> <strong><em>Main interests (click to expand):</em></strong> <br />
Protein structure and probabilistic models. <br />
</summary>
<br />
My main goal in research is to use Machine Learning to gain new insights into biological problems in general and protein structure in particular. In my current project, we predict 3D protein structure along with the uncertainty over all pairwise distances between atoms.
</details>

</br> | | | | | |
:-:|:-:|:-:|:-:|:-:|:-:|
| ![](/social/half_filler.png) |[![](/social/twitter.png)](https://twitter.com/ArtsMarloes) | [![](/social/email.png)](mailto:ma@di.ku.dk) |[![](/social/github.png)](https://github.com/mearts)  | [![](/social/linkedin.png)](https://www.linkedin.com/in/marloes-arts) | ![](/social/half_filler.png) |

<br /><br />

<h4 id="sebastian"></h4>

![](/images/Sebastian.png) <br />
#### Sebastián García López<br><span style="font-weight:normal">(PhD student)</span>
<strong><em>Main interest:</em></strong> <br />
Protein sequence-to-structure relationship,</br>
deep probabilistic modelling for protein representation.

</br> | | | | | |
:-:|:-:|:-:|:-:|:-:|:-:|
| ![](/social/half_filler.png) |![](/social/filler.png) | [![](/social/email.png)](mailto:Sebastian.Garcia.Lopezs@di.ku.dk) |[![](/social/linkedin.png)](https://www.linkedin.com/in/sebastian-garcia-lopez-677680125/)  | ![](/social/filler.png) | ![](/social/half_filler.png) |

<br /><br />

<h4 id="frederikke"></h4>

![](/images/Frederikke.png) <br />
#### Frederikke Isa Marin<br><span style="font-weight:normal">(industrial PhD student, shared between KU & <a href="https://www.novozymes.com/en">Novozymes</a>)</span>
<details>
<summary><strong><em>Main interests (click to expand):</em></strong> <br />
Modelling DNA and protein sequences using machine learning. </br>
Deep latent variable models and representation learning. </br>
</summary>
<br />
Although I have mainly worked with proteins my PhD project (in collaboration with <a href="https://www.novozymes.com/en">Novozymes</a>) will focus on deciphering the genetic code to predict what regions of DNA becomes a protein products. The work will also explore how to generalize across very diverse genomic sequences that are only distantly evolutionarily related and provide meaningful representations of these relationships.
</details>

</br> | | | | | |
:-:|:-:|:-:|:-:|:-:|:-:|
| ![](/social/half_filler.png) |[![](/social/twitter.png)](https://twitter.com/FIMarin42) | [![](/social/email.png)](mailto:f.marin@di.ku.dk) |[![](/social/github.png)](https://github.com/frederikkemarin)  | [![](/social/linkedin.png)](https://www.linkedin.com/in/frederikke-isa-marin-08640413b) | ![](/social/half_filler.png) |

<br /><br />

<h4 id="richard"></h4>

![](/images/Richard1.png) <br />
#### Richard Michael<br><span style="font-weight:normal">(PhD student)</span>
<details>
<summary> <strong><em>Main interests (click to expand):</em></strong> <br />
Probabilistic/Bayesian Machine Learning,<br />Probabilistic Modeling. <br />
</summary>
<br />
In research, my main motivation is gaining an in depth understanding and finding solutions for bioinformatics questions. I'm interested in finding models with insightful analytical properties and degrees of uncertainties. My current project is about predictive modeling using Gaussian Processes on Protein Variants for stability and protein mutations under stress.
</details>

</br> | | | | |
:-:|:-:|:-:|:-:|:-:|
| ![](/social/filler.png) |[![](/social/github.png)](https://github.com/RMichae1) |[![](/social/website.png)](http://laplaceml.com/)  | [![](/social/linkedin.png)](https://www.linkedin.com/in/rimichael/) | ![](/social/filler.png) |

<br /><br />

<h4 id="laurits"></h4>

![](/images/laurits.png) <br />
#### Laurits Fredsgaard Larsen<br><span style="font-weight:normal">(student)</span>
<details>
<summary> <strong><em>Main interests (click to expand):</em></strong> <br />
</summary>
<br />
My focus during the my master's degree in Molecular Biomedicine was on
designing and producing proteins for making modular virus like particle
vaccines.
I am now particularly interesting in using deep learning and generative models
to guide intelligent protein design.
</details>

</br> | | | | |
:-:|:-:|:-:|:-:|:-:|
| ![](/social/filler.png) |[![](/social/github.png)](https://github.com/lauritsf) |[![](/social/website.png)](http://laurits.me/) | [![](/social/linkedin.png)](https://www.linkedin.com/in/lauritsfredsgaard/) | ![](/social/filler.png) |


<br /><br />

# Alumni

<h4 id="maher"></h4>

![](/images/Maher.png) <br />
#### Maher Kassem<br><span style="font-weight:normal">(previously postdoc, currently data scientist at  <a href="https://www.novozymes.com/en">Novozymes</a>)</span>
<strong><em>Main interests:</em></strong> <br />
Previously protein structure & protein stability.<br />
Currently ML and software development, digital products.

</br> | | | | | |
:-:|:-:|:-:|:-:|:-:|:-:|
| ![](/social/half_filler.png) |[![](/social/twitter.png)](https://twitter.com/MaherKassem90) | [![](/social/email.png)](mailto:mahermkassem@gmail.com) |[![](/social/github.png)]( https://github.com/mahermkassem)  | [![](/social/linkedin.png)]( linkedin.com/in/maher-m-kassem-848061b1) | ![](/social/half_filler.png) |

<br /><br />

<h4 id="tone"></h4>

![](/images/Tone.png) <br />
#### Tone Bengtsen<br><span style="font-weight:normal">(previously postdoc, shared between KU & <a href="https://www.novozymes.com/en">Novozymes</a>)</span>
<strong><em>Main interests:</em></strong> <br />
Protein design and how to incorporate protein dynamics into machine learning models.

</br> | | | | | |
:-:|:-:|:-:|:-:|:-:|:-:|
| ![](/social/half_filler.png) |[![](/social/twitter.png)](https://twitter.com/BengtsenTone) | [![](/social/email.png)](mailto:tone.bengtsen@di.ku.dk) |[![](/social/github.png)](https://github.com/tbengtsen)  | [![](/social/linkedin.png)](https://www.linkedin.com/in/tone-bengtsen?originalSubdomain=dk) | ![](/social/half_filler.png) |

<h4 id="jacob"></h4>

![](/images/Jacob.png) <br />
#### Jacob Kæstel-Hansen<br><span style="font-weight:normal">(previously master student)</span>
<strong><em>Main interest:</em></strong> <br />
Mapping mutational effects on proteins using machine learning.

</br> | | | | | |
:-:|:-:|:-:|:-:|:-:|:-:|
| ![](/social/half_filler.png) |![](/social/filler.png) | [![](/social/email.png)](mailto:bzx569@alumni.ku.dk) |[![](/social/linkedin.png)](https://dk.linkedin.com/in/jacob-k%C3%A6stel-hansen-502008115)  | ![](/social/filler.png) | ![](/social/half_filler.png) |

<br /><br />
